src/main.rs

```diff
@@ -647,13 +647,22 @@ fn bisect_nightlies(cfg: &Config, client: &Client) -> Result<BisectionResult, Er
     let dl_spec = DownloadParams::for_nightly(&cfg);
     let now = chrono::Utc::now();
     let today = now.date();
-    let mut nightly_date = today;
+    let mut nightly_date = if let Some(Bound::Date(date)) = cfg.args.start {
```

Comment:
I think these are reverse from what I would expect (or at least inconsistent with `bisect_ci`)... `start` is the "earliest" point and `end` is the "latest" point as implemented there, so this should be `cfg.args.end` and `end_at` should be `cfg.args.start`. Maybe we should switch them though...

Reply:
--help says the end point should pass the test and the start point should *not* pass the test. That would be the opposite of what you're saying bisect_ci is doing.

Reply:
I don't think so? The start point should compile, therefore not passing the test, whereas the end point shouldn't compile therefore passing the test. I think @kennytm's comment (https://github.com/rust-lang-nursery/cargo-bisect-rustc/pull/5#issuecomment-375505005) also supports my reasoning, though I'm not 100% sure.

(it might be easier to discuss this on IRC, feel free to ping me if you want in #rust-infra as simulacrum).